api/src/routes/settings.ts

```diff
@@ -31,10 +31,10 @@ app.post("/v0/settings", cors(), async (ctx) => {
 
   const parsedContent = SettingsSchema.parse(content);
   // Remove the stored api key if the feature is disabled
-  if (!parsedContent.aiEnabled) {
-    parsedContent.openaiApiKey = undefined;
-    parsedContent.anthropicApiKey = undefined;
-  }
+  // if (!parsedContent.aiEnabled) {
```

Comment:
i'd say either remove this chunk or uncomment

part of the logic behind this was to allow the end user to delete api keys... but i don't think that worked anyhow